### ISSUE TITLE
NUnitLite NuGet package no longer installs NUnit NuGet package

### ISSUE THREAD

**Issue by ChrisMaddock** [nunit/nunit#2299]
The NUnit Lite package no longer installs the NUnit Framework package as a dependency.

We added empty dependency groups to the nuspec, to fix https://github.com/nunit/nunit/issues/2165 - but missed that the NUnitLite package _should_ have a dependency on all platforms of the NUnit package. 😞 

![image](https://user-images.githubusercontent.com/4837132/27834023-27ac41ec-60cd-11e7-8505-76314537810b.png)

**Comment by CharliePoole** [comment]
When did the dependency get dropped? Can we fix it at the package level without a new release? I'd call this critical.

**Comment by ChrisMaddock** [comment]
This would have been broken since 3.7.0, which has been released over a month now. I guess there's not that many people adding NUnitLite to new projects!

I debated 'critical' but went for 'pri:high' as a) it's not been reported for a month and b) there's a simple workaround. I don't think we can fix it without a release, no.

**Comment by CharliePoole** [comment]
Too bad NuGet doesn't support a way to do package releases containing the same software. Apt does that with a special suffix.

**Comment by rprouse** [comment]
Odd, NUnit is still listed as an overall dependency and we still have the blank 

```xml 
      <group>
          <dependency id="NUnit" version="[$version$]" />
      </group>
```

I guess that only applies to not specified targets and we need to duplicate it for each?

We could grab the binaries out of the existing NuGet package and release an updated version with the dependencies fixed using the same binaries. I might even be able to edit the existing one using NuGet package manager for a hotfix.

**Comment by rprouse** [comment]
@ChrisMaddock I added myself as a second assignee on this. Let's coordinate. Maybe you can do the proper fix for the next release and I can see if I can get a hotfix out?

**Comment by CharliePoole** [comment]
I wondered about the use of $version$ in other places in the file besides the primary version entry for the package. Are you sure that works as a general substitution?

**Comment by rprouse** [comment]
@CharliePoole that dependency hasn't changed and it looks right in the image @ChrisMaddock posted above. Notice the first dependency is correct, it just doesn't seem to get picked up because of the target specific dependencies.

**Comment by CharliePoole** [comment]
Seems like a bug in NuGet, which we will have to work around.

**Comment by ChrisMaddock** [comment]
@rprouse Sounds good. On my phone, but there's a PR from yesterday for the in-repo fix.

**Comment by rprouse** [comment]
I really don't want to put out a framework hotfix just for this, so I edited the existing NUnitLite package and fixed the dependencies. It won't let me overwrite the 3.7.1 version, so I updated the version of the NuGet package only to 3.7.2. The binaries and NUnit dependency are still 3.7.1.

This might be confusing for some users, so I would like to limit the time this is out there and get the 3.8 update of the framework out on time next month or possibly early.
